schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
 Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
+framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established a framework agreement. The suppliers on the framework are listed in the award section of the related process.
```

Comment:
```suggestion
framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established the framework agreement. The suppliers on the framework are listed in the award section of the related process.
```
the 2nd sentence refers to "the" framework so I think the first sentence should too

Reply:
I might be wrong, but the second sentence uses 'the' because it refers back to the framework agreement introduced in the first sentence. Whereas the first sentence uses 'a' because it is introducing the framework agreement for the first time.

Reply:
hmm, yes okay I think that makes sense...